src/workspace.h

```diff
@@ -1,7 +1,6 @@
 #ifndef WORKSPACE_H
 #define WORKSPACE_H
 
-#include <QDebug>
```

Comment:
If this were to be avoided, I think the PR looks relatively  alright

Reply:
Yeah, on my system this was picking up the Qt5 header instead of the Qt6 one somehow and causing all sorts of issues. That speaks to there probably being some build environment screwup on my end, but I didn't want to fight with CMake to sort that out and I determined it would be easier to remove QDebug. I can try a little harder to get that working tonight.